src/main/java/org/broadinstitute/hellbender/utils/io/Resource.java

```diff
@@ -9,6 +9,7 @@
 public final class Resource {
     private final String path;
     private final Class<?> relativeClass;
+    public final static String LARGE_RESOURCES_PATH = "large";
```

Comment:
Add a comment here explaining what this is

Reply:
Done.